### PR TITLE
Product_card

### PR DESCRIPTION
**Pull request by yurii-krymskiy**
- [DEMO LINK](https://yurii-krymskiy.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://yurii-krymskiy.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="home-page__content">
+
+    <div data-qa="card" class="card">
+      <img src="images/imac.jpeg" alt="iMac" class="card__image">
+
+      <p class="card__text-1">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__text-2">Product code: 195434</p>
```
Comment (IvanFesenko) [inline src/index.html:20]: What does -1, -2 means?
Use logical class names
```suggestion
      <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
      <p class="card__code">Product code: 195434</p>
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,42 @@
...
+
+      <div class="card__block-1">
+        <div class="stars stars--1">
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+          <div class="stars-star"></div>
+        </div>
```
Comment (IvanFesenko) [inline src/index.html:29]: Stars should work with any modifiers (1-5)

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__text-1 {
+  line-height: 18px;
+  width: 166px;
```
Comment (IvanFesenko) [inline src/style.css:29]: Do not hard code width

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
```
Comment (IvanFesenko) [inline src/style.css:33]: move repeatable colors to variables

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+  margin: 0 16px 4px;
+}
+
+.card__text-2 {
+  line-height: 14px;
+  width: 98px;
```
Comment (IvanFesenko) [inline src/style.css:39]:
```suggestion
```

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Pay attention, that your tests are failed.
Also check my comments

File: src/index.html (hunk 2 above)
Comment (yurii-krymskiy) [reply src/index.html:29]: What do you mean here?

**Pull request closed without merging**